Zend/zend_builtin_functions.h

```diff
@@ -22,12 +22,12 @@
 
 #include "zend_types.h"
 
-typedef struct _zval_struct zval;
+struct _zval_struct;
 
 zend_result zend_startup_builtin_functions(void);
 
 BEGIN_EXTERN_C()
-ZEND_API void zend_fetch_debug_backtrace(zval *return_value, int skip_last, int options, int limit);
+ZEND_API void zend_fetch_debug_backtrace(struct _zval_struct *return_value, int skip_last, int options, int limit);
```

Comment:
Usage of ``struct _zval_struct`` instead of ``zval`` looks terrible for me.

Note that duplicate declarations were introduced by previous "header clenup" patches (like 6335264c07b9a73699fe47dc71a452a4ac85c22c). Now you remove them and make sources even worse...